### PR TITLE
Prepare transformers for v0.8.0 huggingface-hub release

### PR DESCRIPTION
**Pull request by LysandreJik**
Updates the staging endpoint to use `hub-ci` instead of `moon-staging`. 

This should be merged only once v0.8.0 is released.

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
Thanks for fixing!

File: setup.py [hunk 1]
```diff
@@ -115,7 +115,7 @@
     "fugashi>=1.0",
     "GitPython<3.1.19",
     "hf-doc-builder>=0.3.0",
-    "huggingface-hub>=0.1.0,<1.0",
+    "huggingface-hub==0.8.0.rc2",
```
Comment (LysandreJik) [inline setup.py:118]: This (and the change in the dependency version table) are isolated in a single commit that will need to be reverted before merging

**Review by LysandreJik — COMMENTED** [review]
@sgugger could you take a second look at this now that it's fully ready?

**Review by sgugger — APPROVED** [review]
Thanks a lot for updating those!

**Pull request merged**